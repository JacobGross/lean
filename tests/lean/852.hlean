import algebra.category.functor.equivalence

-- print prefix category.equivalence

check @category.equivalence.to._Fun
