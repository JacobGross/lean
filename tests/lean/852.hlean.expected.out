category.equivalence.to._Fun :
  Π {C : category.Precategory} {D : category.Precategory} (c : category.equivalence C D)
  {a b : category.Precategory.carrier C},
    category.hom a b →
    category.hom (functor.to_fun_ob (category.equivalence.to_functor c) a)
      (functor.to_fun_ob (category.equivalence.to_functor c) b)
